### PR TITLE
IRIS | Bump opt-in modal to 1.0.31

### PR DESCRIPTION
**Pull request by xkxd**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -103,5 +102,8 @@
     "sinon": "1.17.3",
     "svgo": "0.6.1",
     "through2": "0.6.5"
+  },
+  "dependencies": {
+    "@wikia/tracking-opt-in": "1.0.31"
```
Comment (rogatty) [inline package.json]: Isn't it a dev dependency if we include the files in the concatenated build? We don't need source files on prod.

Comment (hakubo) [reply package.json]: hmm I have just merged
https://github.com/Wikia/discussions-front-end/pull/206/files
where it's also dependency
ok?

Comment (xkxd) [reply package.json]: true, I've just npm installed this without flag

Comment (rogatty) [reply package.json]: @hakubo it doesn't make any practical difference, it's just for clarity.

**Pull request merged**